src/pages/Main/ShopListContainer/ShopListContainer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ListInfo from '../ListInfo/ListInfo';
+import SlideButton from '../SlideButton/SlideButton';
+import './ShopListContainer.scss';
+
+class ShopListContainer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      listData: [],
+      isLeftSide: true,
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/listInfoData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          listData: data,
+        });
+      });
+  }
+
+  moveImg = () => {
+    this.setState({
+      isLeftSide: !this.state.isLeftSide,
+    });
+  };
+
+  render() {
+    const { listData, isLeftSide } = this.state;
+    return (
+      <div className="shopListContainer">
+        <span className="containerTitle">믿고 보는 맛집 리스트</span>
+        <div className="rowContainer">
+          <div className={isLeftSide ? 'filterListLeft' : 'filterListRight'}>
+            {listData.map((data, idx) => {
+              return <ListInfo info={data} key={idx} />;
+            })}
+          </div>
+        </div>
+        <SlideButton click={this.moveImg} rightButton={this.state.isLeftSide} />
```

Comment:
이 부분 구조분해할당 빼먹으신 것 같네요!
그리고 rightButton인데 반대되는 isLeftSide라는 명칭과 사용은 혼란을 줄 수 있을 것 같습니다.

```suggestion
        <SlideButton click={this.moveImg} isLeftBtn={isLeftBtn} />
```

Reply:
감사합니다.
버튼 컴포넌트도 구조분해 함께 하고
명칭도 수정했습니다!